src/style.css

```diff
@@ -1,3 +1,79 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  height: 60px;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__link {
+  display: block;
```

Comment:
you don't need to set `display: block`  because it is `block` by default in this context